common/config/.golangci.yaml

```diff
@@ -49,6 +50,18 @@ linters-settings:
   gocyclo:
     # minimal cyclomatic complexity to report
     min-complexity: 15
+  depguard:
+    rules:
+      main:
+        allow:
```

Comment:
seems you should remove the allow, otherwise it will treat unlisted package disallowed